packages/webapp/src/containers/Consent/locales/es/Owner.Consent.md

```diff
@@ -0,0 +1,339 @@
+
+- Información del registro de cosecha: campo cosechado; cosecha ya cosechada; cantidad.
+
+- Información del registro de siembra: campo sembrado; cultivo sembrado; profundidad de espaciado, largo, ancho y proporción.
+
+- Información del registro de trabajo de campo: campo; tipo de trabajo (arado, labrado en cumbrera, labrado zonal, labranza del mantillo, arar).
+
+- Información de registro de análisis de suelo: campo; profundidad; textura; porcentaje de potasio, fósforo, nitrógeno, materia orgánica, carbono orgánico, carbono inorgánico, carbono total, azufre, calcio, magnesio, sodio, zinc, manganeso, hierro, cobre, boro; capacidad de intercambio catiónico (CEC); grado de acidez (pH); densidad de granel.
+
+- Información del registro de riego: campo; tipo de riego (goteo, aspersión, subsuelo, inundación); tasa de flujo; tiempo total de riego.
+
+- Información del registro de exploración: campo; cosecha; tipo (cosecha, plaga, enfermedad, maleza, otro); acción necesaria (sí/no).
+
+- Otra información de registro: campo; cultivos.
+
+- Información de los turnos: fecha; duración del turno; horas de inicio y finalización; duración de la pausa; actividad de turno (preparación de la cama de cultivo, entrega, ventas, siembra, cosecha, fertilización, ventas, exploración, control de plagas, lavado y empaque, deshierbe, otras tareas personalizadas); asignación de tiempo a cada actividad (por campo o cultivo); estado de ánimo (feliz, muy feliz, triste, muy triste, neutral).
+
+- Información de gastos: fecha; tipo (equipo, fertilizante, combustible, maquinaria, semillas, pesticidas, tierra); nombre; monto del gasto; asociación con cultivos
+
+- Información de ventas: fecha; nombre del comprador/mercado; cosecha; cantidad vendida (kg); ganancias ($)
+
+- Información de registro de siembra: campo sembrado; cultivo sembrado; profundidad de espaciado, largo, ancho y proporción
+
+- Datos del trabajador: número de trabajadores registrados; salario/pago
+
+- Detalles de la unidad: métrico o imperial
+&nbsp;
+
+&nbsp;
+
+
+**Datos de uso**
+&nbsp;
+
+&nbsp;
+
+
+Recopilamos datos de uso (es decir, estadísticas generales sobre los(as) usuarios(as), patrones de tráfico y cómo los(as) usuarios(as) responden a las diversas funciones del sitio) para mejorar la calidad de los servicios que brindamos. Este tipo de datos incluye: cuando Ud. inicia sesión en LiteFarm, las páginas que solicita, cuánto tiempo visita el sitio, cómo interactúa con la aplicación y detalles técnicos sobre su dispositivo (por ejemplo, navegador, tipo de pantalla y procesador). Estos datos se asociarán con su nombre de usuario en LiteFarm (si ha iniciado sesión en la aplicación) o su dirección IP (si no ha iniciado sesión). Podríamos utilizar software como Google Analytics para recopilar estos datos, pero en estos casos, toda la información personal será anónima.
+&nbsp;
+
+&nbsp;
+
+
+** ¿Cómo utilizamos los datos? **
```

Comment:
Markdown doesn't seem to be processing this line properly. I believe removing the spaces between the question mark characters and the asterisks should fix it.